.changeset/wicked-clocks-march.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
I would call this at least a patch, since it will have a user-facing change in [one of our targets](https://khan.github.io/perseus/). Not a blocker though.

Reply:
@handeyeco I don't see any user-facing changes in production code (except adding comments to props - which I guess isn't user-facing).   The link you shared only goes to our Github-hosted storybook.  I feel I'm missing your point.

Reply:
My point is just that Storybook _is_ one of our targets; it's a UI that people see. I think empty changesets should be used very rarely and for changes that are completely invisible to all UIs. Just a personal opinion.

Reply:
I think we have differing views of what Changesets should cover. I see changesets as a way to communicate with folks who integrate Perseus into a project. Storybook is an internal tool and I see very little value in writing changesets to document UI changes to storybook (who would read them and what value would they provide over the PR and it's description?).